flixel/FlxObject.hx

```diff
@@ -1292,11 +1292,11 @@ class FlxObject extends FlxBasic
 
 		var rect = getBoundingBox(camera);
 		var gfx:Graphics = beginDrawDebug(camera);
-		drawDebugBoundingBox(gfx, rect, allowCollisions, immovable);
+		drawDebugBoundingBox(gfx, rect, allowCollisions, immovable, camera.zoom);
 		endDrawDebug(camera);
 	}
 
-	function drawDebugBoundingBox(gfx:Graphics, rect:FlxRect, allowCollisions:Int, partial:Bool)
+	function drawDebugBoundingBox(gfx:Graphics, rect:FlxRect, allowCollisions:Int, partial:Bool, ?zoom:Float = 1.0)
```

Comment:
I think it's fine to simply use a size of 0.5 like it has been, in fact, there's benefits to having this not perfectly align with the camera's pixel resolution, as it more discernible, plus adding new args may break overriding classes

Reply:
Yeah... I don't think you would want that, I didn't want to do it either, but without doing a "zoom" variable, this is what it looks like:
![imagen](https://github.com/HaxeFlixel/flixel/assets/84867412/a48276fa-0373-48c6-bd6a-50f8e42405c3)

Compared to this:
![imagen](https://github.com/HaxeFlixel/flixel/assets/84867412/a2fc4e61-4dcb-40e3-a72d-e6645a252af9)

This is normal btw, this is OpenGL doing its shenanigans and sometimes its not 100% accurate to display stuff in exchange of performance.

One thing that I tried as possible to make it not break a lot, is adding a "Nulled" already set variable at the end, so people with older code and trying to port it could use this function and would not give an error

Reply:
I was able to omit the zoom arg, without missing lines (I didn't see any at zoom = 0.3, but I see a little at zoom=0.2, which I'm okay with)